### ISSUE TITLE
fork of `react-native-web`?

### ISSUE THREAD

**Issue by piranna** [react-native-web-community/contributing#2]
Since maintenance of `react-native-web` is really low and they are very exhitant to accept pull-requests, what about doing here a community managed fork of `react-native-web` so we could improve it easier?

**Comment by Minishlink** [comment]
Forking a still living package is rarely a good way to maintain it better:
- it's not helping the original module and author
- it divides the community
- it can create more problems than it solves due to a lack of competence (we don't know the codebase as well as the original author)

What we need to do here is analyze the situation. You wrote about "really low maintenance" and "very hesitant to accept pull-requests". Can you give examples? It'll be easier to see eventual problems, trace back the root cause and find solutions.

**Comment by piranna** [comment]
I didn't give you an answer before because it's not possible to have proves of the deleted comments... but that's not the case of the pull-request: implementation of the missing StatusBar API and it was rejected because [it's not useful](https://github.com/necolas/react-native-web/pull/850#issuecomment-378057037). That's absurd.

**Comment by necolas** [comment]
> "really low maintenance"

This is not true. There have been over 150 releases and the project was compatible with React 16 within a day of it's release.

> "very hesitant to accept pull-requests"

I accept pull requests. You didn't like that I didn't accept yours.
